pages/common/fmt.md

```diff
@@ -0,0 +1,19 @@
+# fmt
+
+> Fills and joins lines from a file to produce output text of width (at most) a given number of characters (75 by default).
+
+- Reformat a file:
+
+`fmt {{file}}`
```

Comment:
Could `file` be `path/to/file` please? This is just to keep it consistent with other pages we have already :-)